redfish_client/resource.py

```diff
@@ -42,6 +42,23 @@ def __init__(self, connector, oid=None, data=None):
             self._content = data
             self._headers = {}
 
+        if isinstance(self._content, dict):
+            for key in self._content.keys():
+                if key.startswith("#") and "target" in self._content[key]:
+                    self._add_action_method(key)
+
+    def _add_action_method(self, action_name):
+        action = action_name.split(".")[-1].lower()
```

Comment:
I think each action has element "Name" or something. Can we use that instead parsing the id?

Reply:
It has element "title", but documentation lacks this element on some actions.